src/BugsnagUnity.m

```diff
@@ -528,6 +540,25 @@ void bugsnag_setEnabledBreadcrumbTypes(const void *configuration, const char *ty
       }
 }
 
+void bugsnag_setEnabledTelemetryTypes(const void *configuration, const char *types[], int count){
+    if(types == NULL)
+    {
+        return;
+    }
+
+    ((__bridge BugsnagConfiguration *)configuration).telemetry &= ~BSGTelemetryInternalErrors;
+    for (int i = 0; i < count; i++) {
+        const char *enabledType = types[i];
+        if (enabledType != nil) {
+            NSString *typeString = [[NSString alloc] initWithUTF8String:enabledType];
+            if([typeString isEqualToString:@"InternalErrors"])
+            {
+                ((__bridge BugsnagConfiguration *)configuration).telemetry |= BSGTelemetryInternalErrors;
```

Comment:
This differs from the Android behaviour, where unknown telemetry options will be disabled.

Maybe make it match like so?

```suggestion
    ((__bridge BugsnagConfiguration *)configuration).telemetry = 0;
    for (int i = 0; i < count; i++) {
        const char *enabledType = types[i];
        if (enabledType != nil) {
            NSString *typeString = [[NSString alloc] initWithUTF8String:enabledType];
            if([typeString isEqualToString:@"InternalErrors"])
            {
                ((__bridge BugsnagConfiguration *)configuration).telemetry |= BSGTelemetryInternalErrors;
```